### ISSUE TITLE
Feature Request / Joybus support for the switch release

### ISSUE THREAD

**Issue by Sorunome** [mgba-emu/mgba#3185]
Hello,
would it be possible to add the Joybus support to link with dolphin also to the nintendo switch builds? that way you could use your switch as a separate screen and controller when emulating e.g. zelda four sword adventures.

Thank you in advance!

**Comment by endrift** [comment]
This would be part of [Orca](https://mgba.io/faq.html#glossary), a low priority feature I've had on the horizon for a while. Netplay and rollback/runahead are important for the foundation of Orca, so I have to wait on those first.
